### PR TITLE
updated 195 vulnerabilities in lunar-core

### PR DESCRIPTION
**Pull request by averyatezcater**

### CODE REVIEW

**Comment by tjwp** [comment]
Should we configure Dependabot for this repo?

File: package.json [hunk 1]
```diff
@@ -39,5 +39,8 @@
   "peerDependencies": {
     "graphql": "^14.0.2",
     "graphql-tools": "^3.1.1"
+  },
+  "dependencies": {
+    "package-lock.json": "^1.0.0"
```
Comment (mjfaga) [inline package.json]: I think this is a mistake?

**Comment by mjfaga** [comment]
@averyatezcater Alternate take at this here for reference: https://github.com/ezcater/lunar-core/pull/13. If you are good with that, I'll close this PR out and merge that one?

**Comment by tjwp** [comment]
@mjfaga dependabot should be setup now

**Pull request closed without merging**